framework/platform/bpl/include/bpl_cfg.h

```diff
@@ -260,6 +265,15 @@ struct BPL_WLAN_PARAMS {
     char security[BPL_SEC_LEN];
 };
 
+/* DCS Parameters */
+typedef struct BPL_DCS_PARAMS {
+    uint8_t enable;
+    uint16_t interval_sec;
+    uint8_t dwell_msec;
+    uint64_t refresh_sec;
+    char channel_pool[BPL_DCS_CHANNEL_POOL_LEN];
```

Comment:
Why is it `char` and not `uint8_t`?

Reply:
BPL_DCS_PARAMS should be removed , it unused and not related to DCS 
feature we are working on.

Reply:
done